docker-compose.yml

```diff
@@ -1,22 +1,24 @@
 version: '2'
 
 services:
+  mongodb:
+    image: 'bitnami/mongodb'
+    # volumes:
+    #   - 'mongodb_data:/bitnami'
```

Comment:
I would remove this for now if it is not necessary.

Reply:
👍